### ISSUE TITLE
Splitting set without duplicating it too produces no result

### ISSUE THREAD

**Issue by ths84** [UB-Mannheim/GTCheck#10]
If you try to split a gt set while leaving "duplicate set for multiple keying" = 0, nothing will happen and GTCheck will not split the gt set.

![Screenshot 2021-09-15 094633](https://user-images.githubusercontent.com/89008142/133392345-53c9a375-f8f7-4666-a29d-99b91ba051e1.png)

It would be more user-friendly to automatically set "duplicate set for multiple keying" if a user chooses to split a set for multiuser keying by the chosen amount of split-parts.

**Comment by JKamlah** [comment]
So thats on purpose. I think the original record should not be able to be split through this interface unless you want to create a duplicate at the same time. But i am open for reasons why we need this option here.

**Comment by ths84** [comment]
Agreed. But then I don't understand the explanation "Splitting creates at least one duplicate" right under the section-header, because if you don't enter a value for duplication as well (that means, leave "number of duplicates" = 0), there will be no duplicates created at all. 

Maybe it would be helpful for users to change the sub-heading or, as I mentioned, to automatically set a value for duplication, if a user trys to split a set.
